compositor_chromium/chromium_sys/build.rs

```
@@ -0,0 +1,163 @@
+        dir::copy(resources_path, &lib_path, &options)?;
+    }
+
+    Ok(bindings)
+}
+
+#[cfg(target_os = "macos")]
+fn link(cef_root: &Path, _target_path: &Path) {
+    let build_type = if cfg!(debug_assertions) {
+        "Debug"
+    } else {
+        "Release"
+    };
+
+    let dst = cmake::Config::new("CMakeLists.txt")
+        .define("MAKE_BUILD_TYPE", build_type)
+        .define("CEF_ROOT", cef_root.display().to_string())
+        .build();
+
+    println!("cargo:rustc-link-search={}", dst.display());
+    println!("cargo:rustc-link-lib=static=cef_dll_wrapper");
+}
+
+#[cfg(target_os = "linux")]
+fn link(cef_root: &Path, target_path: &Path) {
+    println!(
+        "cargo:rustc-link-search=native={}",
+        PathBuf::from(cef_root).join("Release").display()
+    );
+    println!(
+        "cargo:rustc-link-search=native={}",
+        target_path.join("lib").display()
+    );
+    println!("cargo:rustc-link-lib=dylib=cef");
+}
+
+#[derive(Debug)]
+struct RemoveCommentsCallback;
+
+impl ParseCallbacks for RemoveCommentsCallback {
+    fn process_comment(&self, _comment: &str) -> Option<String> {
+        Some(String::new())
+    }
+}
+
+fn download_cef(cef_root_path: &Path) {
+    let platform = if cfg!(target_os = "macos") {
+        if cfg!(target_arch = "arm") {
+            "macosarm64"
+        } else {
+            "macosx64"
+        }
+    } else if cfg!(target_os = "linux") {
+        "linux64"
+    } else {
+        panic!("Unsupported platform");
+    };
+
+    let download_path = cef_root_path.parent().unwrap();
+    let url = format!("https://cef-builds.spotifycdn.com/cef_binary_115.3.11%2Bga61da9b%2Bchromium-115.0.5790.114_{platform}_minimal.tar.bz2");
+    let resp = reqwest::blocking::get(url).unwrap();
+
+    let archive_name = "cef.tar.bz2";
+    let content = resp.bytes().unwrap();
+    fs::write(&download_path.join(archive_name), content).unwrap();
+
+    let _ = fs::create_dir_all(&cef_root_path);
```

don't ignore errors (it's fine to unwrap, but it shouldn't be silently ignored)